test/ava-files.js

```diff
@@ -104,3 +104,29 @@ test('findFiles - does not return duplicates of the same file', function (t) {
 		t.end();
 	});
 });
+
+test('findFiles - finds the correct files by default', function (t) {
+	var avaFiles = new AvaFiles(['**/ava-files/default-patterns/**']);
```

Comment:
Argh. I'm realizing this is actually kinda hard to test.

This will actually find every file in the `default-patterns` folder.

Instead of this argument, can you just `process.chdir()` to the `default-patterns` folder, and drop the constructor argument?

You are technically overriding the default here.

#863 introduce `cwd` and `resolveTestsFrom` options, that would make this doable without changing the directory.